src/worker/exporters/node_exporter.py

```diff
@@ -0,0 +1,161 @@
+######################################################
+# This exporter file can be modified or used as
+# an example of how to create a custom exporter.
```

Comment:
Should we also add a high level overview of what this example exporter does? And, then highlight that a user can just edit the relevant data collection/parsing part to get custom metrics?

Reply:
Like with inline comments?

Reply:
I was thinking like an overview comment as well as some inline comments in specific section.

Reply:
I'll address this

Reply:
Alright I have added additional inline comments to the exporter code and an overview of the Node exporter to the documentation.